documentation/modules/security/con-certificate-renewal.adoc

```diff
@@ -26,7 +39,30 @@ Not Before                                     Not After
                               <--- renewalDays --->|
 ----
 
-The behavior of the Cluster Operator during the renewal period depends on whether the relevant setting is enabled, in either `Kafka.spec.clusterCa.generateCertificateAuthority` or `Kafka.spec.clientsCa.generateCertificateAuthority`.
+To make a change to the validity and renewal periods after creating the Kafka cluster, you configure and apply the `Kafka` custom resource,
+and xref:proc-renewing-ca-certs-manually-{context}[renew the CA certificates].
```

Comment:
I think you should clarify that if you don't trigger manual renewal it will be used next time the certificate is renewed. Even that might be useful for some users.

Reply:
> If you do not manually renew the certificates, the periods will be used the next time the certificate is renewed automatically.